frontend/src/components/MyNavbar/MyNavbar.test.js

```diff
@@ -0,0 +1,6 @@
+import enzyme, {shallow} from 'enzyme';
+import MyNavbar from './MyNavbar';
+import React from 'react';
+
+const wrapper = shallow(<MyNavbar/>);
```

Comment:
Great job writing tests Adem! Can you make this a little more efficient by checking if there is some text in the navbar. If the text exists, then we know that the navbar has rendered correctly. I can help you with this if youd like.

Reply:
Added the test for 'Recipe Maker' in the Navbar and I will add some more tests for the buttons tomorrow.